libs/wingc/src/jsify.rs

```diff
@@ -1405,7 +1458,16 @@ impl<'a> JSifier<'a> {
 		}
 
 		let func_prefix = prefix.join(" ");
-		let mut code = new_code!(&func_def.span, func_prefix, "(", parameters.clone(), ")", arrow, "{");
+		let mut code = new_code!(
+			&func_def.span,
+			func_prefix,
+			"(",
+			implicit_parameters,
```

Comment:
I am assuming the reason the implicit parameters are before the explicit ones is in order to support variadic arguments, correct?

Worth mentioning

Reply:
Yes, and to look similar to ctors and to the construct pattern in general.